src/helpers/permit.ts

```diff
@@ -57,7 +57,7 @@ export const generatePermit2Signature = async (spender: string, amountInEth: str
 
   const base64encodedTxData = Buffer.from(JSON.stringify(txData)).toString("base64");
 
-  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&chainId=0x${chainId.toString(16)}`;
+  const result = `${permitBaseUrl}?claim=${base64encodedTxData}&network=${chainId.toString(10)}`;
```

Comment:
Do you need to specify base 10?

Reply:
I don't need to do that

Reply:
You should remove that code if it isn't necessary.

Reply:
Removed